css/port.css

```diff
@@ -236,142 +336,325 @@ section .aim ul li {
 }
 
-.about {
-  width: 375px;
-  margin: 70px auto;
-  background-color: #fff;
-  border-top-right-radius: 100px;
-  padding: 100px 0;
-  height: 1263px;
-}
+/* -----aboutus------------ */
+@media (max-width: 470px) {
+  .about {
+    margin-top: -420px;
+  }
 
-.about_header {
-  font-weight: bold;
-  font-size: 40px;
-  line-height: 52px;
-  color: #172b4d;
-  margin-bottom: 10px;
-  margin-top: 0;
-}
+  .about_header {
+    font-weight: 700px;
+    font-size: 40px;
+    line-height: 52px;
+    color: #091e42;
+    height: 114px;
+  }
 
-.about_img {
-  position: relative;
-  bottom: -9px;
+  .left-block {
+    padding-top: 100px;
+    padding-left: 14px;
+  }
+
+  .about_text {
+    width: 400px;
+  }
+
+  .aim {
+    padding-left: 14px;
+    margin-top: 120px;
+  }
 }
 
 .contact_container {
-  padding: 10px 0;
-  height: 850px;
-  border-top-left-radius: 90px;
+  max-width: 100%;
+  height: 846px;
   background-color: #6070ff;
-  margin-top: 3850px;
-  width: 375px;
+  border-top-left-radius: 90px;
+  margin-top: -250px;
+  padding-top: 117px;
 }
 
 .contact_bk {
-  background-image: url('../images/contact\ form\ shapes-mobile.svg');
-  background-repeat: no-repeat;
-  background-size: cover;
-  background-position: top;
+  width: 1290.66px;
+  height: 718.7px;
+  background-image: url('../images/contact-form-background-shapes@2x.png');
+  margin-left: 75.34px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  height: 577px;
+  padding-top: 50px;
+  width: 100%;
+  margin: auto;
 }
 
-.feedback {
-  font-weight: bold;
-  font-size: 35px;
+.contact_header {
+  font-weight: 700px;
+  font-size: 40px;
   line-height: 52px;
-  text-align: center;
   color: #fff;
-  margin-bottom: 0;
-  margin-top: 150px;
 }
 
-.msg {
-  color: #ebebff;
+.contact_p {
+  font-weight: 400px;
   font-size: 20px;
   line-height: 28px;
+  color: #ebebff;
   text-align: center;
-  font-weight: 400px;
-}
-
-.contact_container .btn {
-  margin-top: 10px;
+  width: 569px;
+  padding-top: 20px;
 }
 
-input textarea {
+textarea {
+  max-width: 100%;
+  height: 200px;
   background: #fff;
   padding: 10px 16px;
-  border-radius: 8px;
+  border-radius: 5px;
   border: 1px solid #cfd8dc;
   color: #172b4d;
   font-size: 17px;
+  width: 447px;
 }
 
-.contact_container textarea {
-  max-width: 100%;
-  height: 200px;
-}
-
-.contact_container form {
-  max-width: 400px;
-  margin: 20px auto;
+input {
+  background: #fff;
+  padding: 10px 16px;
+  border-radius: 5px;
+  border: 1px solid #cfd8dc;
+  color: #172b4d;
+  font-size: 17px;
+  width: 447px;
 }
 
-.contact_container textarea input {
-  width: 100%;
-  margin: 10px 0;
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 27px;
+  width: 447px;
+  padding-top: 27px;
+  align-items: center;
 }
 
-.line {
-  background: #0d151e;
-  border-radius: 10px;
-  height: 5px;
-  width: 134px;
-  margin-top: 10px;
-  margin-left: 100px;
+@media (max-width: 470px) {
```

Comment:
- [ ] Kindly remove this media query for the mobile version. Make sure to use media queries for only Desktop breakpoint in the CSS file.